src/js/static-net-filtering.js

```diff
@@ -26,6 +26,7 @@
 /******************************************************************************/
 
 import globals from './globals.js';
+import timer from './timer.js';
```

Comment:
For now only this file uses it, but once all files use it then we can fix `globals.js`.